raiden/network/proxies/token_network.py

```diff
@@ -900,31 +871,6 @@ def _check_why_deposit_failed(
 
         return error_type, msg
 
-    def _close_preconditions(
-            self,
-            channel_identifier: ChannelID,
-            partner: Address,
-            block_identifier: BlockSpecification,
-    ):
-        if not self.client.can_query_state_for_block(block_identifier):
-            raise NoStateForBlockIdentifier()
-
-        self._check_for_outdated_channel(
-            participant1=self.node_address,
-            participant2=partner,
-            block_identifier=block_identifier,
-            channel_identifier=channel_identifier,
-        )
-
-        error_type, msg = self._check_channel_state_for_close(
```

Comment:
Here `_check_for_outdated_channel` and `_check_channel_state_for_close` did the same call to `_detail_channel`. This PR fixes it